platforms/core-configuration/configuration-cache/src/main/kotlin/org/gradle/internal/cc/impl/problems/ConfigurationCacheProblemsSummary.kt

```diff
@@ -235,15 +270,15 @@ class Summary(
                         appendLine("  See ${documentationRegistry.getDocumentationFor(it.page, it.anchor)}")
                     }
                 }
-                if (reportableProblemCauseCount > MAX_CONSOLE_PROBLEMS) {
-                    appendLine("plus ${reportableProblemCauseCount - MAX_CONSOLE_PROBLEMS} more problems. Please see the report for details.")
+                if (uniqueProblemCount > MAX_CONSOLE_PROBLEMS) {
+                    appendLine("plus ${uniqueProblemCount - MAX_CONSOLE_PROBLEMS} more problems. Please see the report for details.")
```

Comment:
❓ Did we change the logic here?
Before, we were checking `reportableProblemCauseCount > MAX_CONSOLE_PROBLEMS`, and then print `"plus ${reportableProblemCauseCount - MAX_CONSOLE_PROBLEMS} more problems."`
Now we do same, but instead of reportableProblems we use uniqueProblems, which includes suppressed silently problems

Reply:
Good catch, thanks! I will add a test and fix this.